rasa_nlu_examples/featurizers/sparse/semantic_map_featurizer.py

```diff
@@ -0,0 +1,690 @@
+import json
+import re
+from collections import defaultdict
+from pathlib import Path
+from typing import Any, Dict, List, Optional, Set, Text, Tuple, Type, Union
+
+import numpy as np
+import scipy.sparse
+from rasa.nlu.components import Component
+from rasa.nlu.constants import (
+    DENSE_FEATURIZABLE_ATTRIBUTES,
+    FEATURIZER_CLASS_ALIAS,
+    TOKENS_NAMES,
+)
+from rasa.nlu.featurizers.featurizer import SparseFeaturizer
+from rasa.nlu.tokenizers.tokenizer import Token, Tokenizer
+from rasa.shared.exceptions import (
+    FileNotFoundException,
```

Comment:
One thing I wonder ... why do we import this exception? Doesn't base python come with `FileNotFoundError`?